vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -111,54 +117,24 @@
         <Annotation Term="Core.Description" String="Granularity of this time dimension is Date" />
         <Property Name="ClosedClosedPeriods" Type="Edm.Boolean" Nullable="false" DefaultValue="false">
           <Annotation Term="Core.Description" String="Application-time periods are stored with closed-closed semantics" />
-          <Annotation Term="Core.LongDescription" String="Use the temporal query option `$toInclusive` instead of `$to` for specifying the period end in time-series queries." />
+          <Annotation Term="Core.LongDescription" String="If `true`, the value of the property referenced by `TimeSliceEnd` is included in the time slice, and the temporal query option `$toInclusive` must be used instead of `$to` for specifying the period end in time-series queries." />
         </Property>

```

Comment:
Name this `EndInclusive`?

Reply:
Even if ClosedClosedPeriods = true in date granularity, can't I use `$from=2021-04-01&$to=2021-05-01` to cover the month of April? The choice between `$to` and `$toInclusive` should be independent of the ClosedClosedPeriods flag. Or: If it is not independent, we don't need `$toInclusive`, we need only `$to`, and whether it is inclusive depends on ClosedClosedPeriods.

ClosedClosedPeriods should only affect how `$at` is handled.

Reply:
You can use `$to` and `$toInclusive` independent of the server-side storage with closed-open or closed-closed periods, the server has to take care of how to translate these into queries to its underlying data store.

The annotation just helps explain the results to the client, and to correctly interpret the values of the `TimeSliceEnd` property.

Reply:
> You can use `$to` and `$toInclusive` independent of the server-side storage with closed-open or closed-closed periods

But the annotation definition says otherwise: If `ClosedClosedPeriod=true` then I must use `$toInclusive`.

Reply:
Thanks, fixed